### PR TITLE
Fixed Errors with spelling, punctuation and some grammar

### PR DESCRIPTION
**Pull request by CleanMachine1**
Hi, I was just reading the README.md as I like this repo and saw a lot of errors within, so I have fixed all I have seen.
Hope you are happy to accept this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-1) download the .deb file for your system architecture from the [latest release](https://github.com/Itai-Nelken/Etcher-arm-32-64/releases/latest) (armhf is 32bit arm, and arm64 is 64bit arm).
-2) open the .deb file with a package installer (just double click it) if you have one installed, or open terminal in the directory where the .deb is and type 
+but if you prefer, you can install it manually: 
+1) download the .deb file for your system architecture from the [latest release](https://github.com/Itai-Nelken/Etcher-arm-32-64/releases/latest) (armhf is 32bit arm and arm64 is 64bit arm).
+2) open the .deb file with a package installer (just double click it) if you have one installed, or open terminal in the directory where the .deb file is and type 
 ```sh
 sudo apt install -y --fix-broken the-file-name.deb
 ```
-but replace `the-file-name.deb` with the name of the .deb file you downloaded (or path to it).
+but replace `the-file-name.deb` with the path and name of the .deb file you downloaded (E.G. /home/$USER/Downloads/etcher.deb).
```
Comment (Itai-Nelken) [inline README.md]: I still prefer the old spelling, but will add a example.

**Comment by Itai-Nelken** [comment]
Thanks a lot!

**Comment by CleanMachine1** [comment]
Nice, even found some your self :)

File: README.md (hunk 1 above)
Comment (CleanMachine1) [reply README.md]: when I was changing that I thought it would be changed by you :+1:

Comment (CleanMachine1) [reply README.md]: I thought you might change that :+1:

**Review by CleanMachine1 — COMMENTED** [review]
Please Ignore, I misclicked and can't find a way to close it

**Pull request merged**